pkgs/applications/blockchains/chia/default.nix

```diff
@@ -45,25 +45,32 @@ python3Packages.buildPythonApplication rec {
     clvm
     clvm-rs
     clvm-tools
+    colorama
     colorlog
     concurrent-log-handler
     cryptography
     dnspython
+    fasteners
     keyrings-cryptfile
     pyyaml
     setproctitle
     setuptools # needs pkg_resources at runtime
     sortedcontainers
+    watchdog
     websockets
   ];
 
   checkInputs = with python3Packages; [
     pytestCheckHook
   ];
 
+  doCheck = false;
```

Comment:
I took the opportunity to make the testsuite optional. It can take half an hour which was slowing down users and Python maintainers considerably, and now it seems to have race condition on less powerful computers like Raspberry Pis.

Reply:
Please add that blurb as a comment right here.

Reply:
done